fog/sql_recovery_db/src/lib.rs

```diff
@@ -2798,4 +2906,164 @@ mod tests {
             .unwrap();
         assert_eq!(accepted_start_block, ingress_key_status.start_block);
     }
+
+    #[test_with_logger]
+    fn test_get_ingress_key_records_should_only_include_in_use_keys_one_key_lost_one_key_retired_and_last_scanned_greater_than_pubkey_expiry(
+        logger: Logger,
+    ) {
+        let mut rng: StdRng = SeedableRng::from_seed([123u8; 32]);
+        let db_test_context = test_utils::SqlRecoveryDbTestContext::new(logger);
+        let db = db_test_context.get_db_instance();
+
+        // At first, there are no records.
+        assert_eq!(
+            db.get_ingress_key_records(
+                0,
+                IngressPublicKeyRecordFilters {
+                    should_include_lost_keys: false,
+                    should_include_retired_keys: false,
+                    should_only_include_unexpired_keys: false,
```

Comment:
Shouldn't this be set to true for this test?

Reply:
It doesn't matter for this first query, since there are no ingress keys in the database at this time. I chose to make this value false because it's the default value in a sense. The assertion later on in the test has this value == true.